js/ListView.js

```diff
@@ -45,8 +41,13 @@ class ListView extends ComponentView {
    * animates the list items in one-by-one
    */
   animateListItems() {
-    this.model.getChildren().forEach((listItem, index) => {
+    const items = this.model.getChildren();
+    const itemCount = items.length;
+    items.forEach((listItem, index) => {
       setTimeout(listItem.toggleActive.bind(listItem, true), 200 * index);
+      if (index !== (itemCount - 1)) return;
+      const $item = this.$('.list-item').eq(index);
+      $item.one('transitionend', () => this.$el.removeClass('is-animating'));
```

Comment:
This isn't as reliable as it sounds.

As [`transition-property`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property) and [`transition-duration`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-duration) can be a comma separated lists and as [transition events are dispatched for each property](https://drafts.csswg.org/css-transitions/#transition-events), multiple [transitionend](https://developer.mozilla.org/en-US/docs/Web/API/Element/transitionend_event) events are fired if a class applies multiple transitions to different properties of different durations. The can mean that a collective transition animation on a single element is not finished on receipt of the shortest property transition duration, if another has a substantially larger duration.